Directory.Build.BeforeCommonTargets.targets

```diff
@@ -2,18 +2,18 @@
   <PropertyGroup>
     <!-- Only build Microsoft.AspNetCore.App, Microsoft.AspNetCore.App.Ref, ref/ assemblies, and ProjectTemplates in source build. -->
     <!-- Analyzer package are needed in source build for WebSDK -->
-    <ExcludeFromSourceBuild
-        Condition="'$(ExcludeFromSourceBuild)' == '' and
-            '$(DotNetBuildFromSource)' == 'true' and
+    <ExcludeFromSourceOnlyBuild
```

Comment:
Much of this is just updates to the controls names to use the new UB names and meanings.